mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -416,6 +438,7 @@ object DenseMatrix {
    * @param numCols number of columns of the matrix
    * @param rng a random number generator
    * @return `DenseMatrix` with size `numRows` x `numCols` and values in U(0, 1)
+   * @since 1.2.0
```

Comment:
It seems that these methods were added to the `DenseMatrix` object in 1.3.0 right?